Source/DafnyCore/Compilers/ExecutableBackend.cs

```diff
@@ -30,10 +30,36 @@ public abstract class ExecutableBackend : IExecutableBackend {
   public override string ModuleSeparator => Compiler.ModuleSeparator;
 
   public override void Compile(Program dafnyProgram, ConcreteSyntaxTree output) {
+    InstantiatePlaceholders(dafnyProgram);
```

Comment:
```suggestion
    InstantiateReplaceableModules(dafnyProgram);
```
One name per concept makes the code easier to read